packages/components/src/components/mobile-dialog/mobile-dialog.tsx

```diff
@@ -119,21 +134,8 @@ const MobileDialog = props => {
                 </Div100vhContainer>
             </div>
         </CSSTransition>,
-        document.getElementById(portal_element_id)
+        document.getElementById(portal_element_id) as HTMLDivElement
```

Comment:
@likhith-deriv Instead of force casting everywhere we can just put it in a variable and then check for the null value once.